drivers/hmis_csv_importer/app/models/hmis_csv_importer/importer/importer.rb

```diff
@@ -0,0 +1,789 @@
+# frozen_string_literal: true
```

Comment:
What is this doing?  (I'm concerned since it's above the copyright, we might break it come 2022.